xlgui/widgets/dialogs.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, parent, title):
 
         self.fields = []
 
-    def add_field(self, label):
+    def add_field(self, label, value=None, tooltip=None):
```

Comment:
probably should document the parameters in the docstring?

Reply:
Done